features/include/pcl/features/organized_edge_detection.h

```diff
@@ -93,14 +93,18 @@ namespace pcl
       void
       compute (pcl::PointCloud<PointLT>& labels, std::vector<pcl::PointIndices>& label_indices) const;
       
-      /** \brief Set the tolerance in meters for difference in depth values between neighboring points. */
+      /** \brief Set the tolerance in meters for difference in depth values between neighboring points.
+        * (The value is set for 1 meter and is adapted with respect to depth value linearly.
+        * (e.g. 2.0*th_depth_discon_ in 2 meter depth)) */
```

Comment:
Can I suggest to rewrite the docstring to.
> If a point is at a 2 meter depth, a neighborhood point is considered discontinuous if their depth difference is larger than  2.0*th_depth_discon_.

Reply:
I am thinking of how to reword the entire getter / setter description.  I don't think "Set the tolerance in meters for difference in depth values between neighboring points" is accurate, as it still sounds like an absolute distance.  I haven't quite figured out the best way to word it yet and am grateful for suggestions.

Reply:
> Set the tolerance in meters for **the relative** difference in depth values between neighboring points.

Explicitly mentioning relative, plus the examples in the lines below would be sufficient for me.